.github/workflows/build.yml

```diff
@@ -55,17 +55,81 @@ jobs:
     #   # echo "${{ secrets.GITHUB_TOKEN }}" | docker login docker.pkg.github.com -u ${{ github.actor }} --password-stdin
     #   run: |
     #     echo ${{ secrets.DOCKER_PASSWORD }} | docker login -u ${{ secrets.DOCKER_USERNAME }} --password-stdin
-  base-notebook:
+    - name: Build Image
+      # Add folder of artifacts related to build that anyone can download
+      run: |
+        # pwd
+        # ls ${PWD}/*
+        python3 build.py --image-prefix=${IMAGE_PREFIX} base-notebook
+  pangeo-notebook:
+    needs: base-notebook
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Checkout Repo
+      uses: actions/checkout@v1
+    - name: Set Up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Enable Pip Cache
+      # https://github.com/actions/cache/blob/master/examples.md#python---pip
+      uses: actions/cache@v1
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-
+    - name: Install Dependencies
+      run: |
+        python --version
+        pip install -r requirements.txt
+        repo2docker --version
+        docker version
+    # - name: Authenticate with DockerHub
```

Comment:
@scottyhq - can you point me in the right direction here?

Reply:
just add DOCKER_PASSWORD DOCKER_USERNAME to Repo Secrets ( https://github.com/pangeo-data/pangeo-stacks/settings/secrets)

and uncomment Authenticate step before any step that pushes to DockerHub. See this line to the bottom: https://github.com/scottyhq/repo2docker-githubci/blob/master/.github/workflows/repo2docker.yml#L66